SMPost/schema-LD.json

```diff
@@ -0,0 +1,175 @@
+{

```

Comment:
schema-LD.json What is the difference with schema.json. guess it is not necessary

Reply:
I understand that in order to use NGSI-LD fully, like, handle multiattribute properties and arrays of relationships etc, you need an example in jsonld. These example-normalized.jsonld files will be handled by schema-LD.json, while key-value payloads will be handled by schema.json.

Reply:
You are right but this is something we expect to launch for all data models 'soon'. So It would be good to test it in the incubated repository (see how to automate it), and then pass it to the official repo. 
Currently is not necessary, but hopefully it will be generated automatically in a coming future.

Reply:
Good to know Alberto, thank you. I will upload in incubated repo when a SocialMedia model folder is created.